library/src/main/res/layout/stream_item_channel.xml

```diff
@@ -81,10 +81,11 @@
             android:background="@color/stream_gray"
             app:layout_constraintBottom_toBottomOf="parent" />
 
-        <TextView
-            android:id="@+id/tv_click"
+        <Button
+            android:id="@+id/btn_wrapper"
```

Comment:
The name is confusing as it doesn't really wrap anything. Maybe `click_area`?
And by the way `Button` could be just replaced with `View`.